server/views/pages/prisons/configuration/config.njk

```diff
@@ -131,12 +72,68 @@
         attributes: { "data-test": "booking-window-edit" }
       })}}
 
-      <hr class="govuk-section-break govuk-section-break--xl govuk-section-break--visible">
+      <hr class="govuk-section-break govuk-section-break--xl">
```

Comment:
Reordering this page for clarity and adding section breaks to space it out a bit.